src/components/visualization/bar-chart/CustomTooltip.tsx

```diff
@@ -0,0 +1,29 @@
+import { DefaultTooltipContent, type TooltipProps } from 'recharts';
+import { computeIfAbsent } from '@/utils/mapUtils';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+interface CustomTooltipProps extends TooltipProps<any, any> {
+  axesMap: Map<string, Map<string, boolean>>;
+  xAxis: string;
+}
+
+export const CustomTooltip: React.FC<CustomTooltipProps> = (props) => {
```

Comment:
Can this be defined as a "normal" React component (`export default function CustomTooltip ...`)?

Reply:
Ah, sure